testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -698,6 +701,18 @@ protected QualifiedObjectName getQualifiedTableName(String tableName)
                 tableName);
     }
 
+    protected String getTableLocation(String tableName)
```

Comment:
The `location` table property is specific to some connectors. I don't think we want to add this method to `AbstractTestQueryFramework`.

Reply:
It looked like nice duplication removal. Other connectors could override that or not use this function. But if this change is not aligned with purpose of `AbstractTestQueryFramework` I will close this PR.